pkg/controller/flow_test.go

```diff
@@ -0,0 +1,182 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller_test
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	scfeatures "github.com/kubernetes-incubator/service-catalog/pkg/features"
+	"github.com/stretchr/testify/assert"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+)
+
+func TestBasicFlowWithBasicAuth(t *testing.T) {
+	// GIVEN
+	ct := NewControllerTest(t)
+	defer ct.TearDown()
+
+	// WHEN
+	ct.CreateSecretWithBasicAuth("user1", "p2sswd")
+	ct.CreateClusterServiceBrokerWithBasicAuth()
+
+	assert.NoError(t, ct.WaitForReadyBroker())
+
+	// THEN
+	ct.AssertOSBBasicAuth(t, "user1", "p2sswd")
+	ct.AssertClusterServiceClassAndPlan(t)
+
+	// uncomment when the https://github.com/kubernetes-incubator/service-catalog/issues/2563 is fixed
+	ct.UpdateSecretWithBasicAuth("user1", "newp2sswd")
+
+	// WHEN
+	ct.CreateServiceInstance()
+
+	// THEN
+	assert.NoError(t, ct.WaitForReadyInstance())
+
+	// uncomment when the https://github.com/kubernetes-incubator/service-catalog/issues/2563 is fixed
```

Comment:
what should be uncommented? Is that Comment still valid?

Reply:
removed